packages/ui-components/src/components/Chart/XAxis.js

```diff
@@ -32,12 +32,13 @@ const X_AXIS_PADDING = {
   },
 };
 
-const renderXAxisLabel = (label, fillColor, isEnlarged) => {
-  if (label && isEnlarged)
+const renderXAxisLabel = (label, fillColor, isEnlarged, isExporting) => {
+  if (label && isEnlarged && !isExporting)
```

Comment:
@tcaiger do you remember why you turned off the x axis label when exporting? Seems like it was very intentional, but it's been recorded as [an issue](https://linear.app/bes/issue/RN-249/xname-aka-x-axis-label-not-exporting)

Easy for me to revert, but I'd bet that means the x axis label will conceal the legend or some other issue...

Reply:
> 

Hey, yes it's just because it was overlapping with the legend and labels. It was very tricky to get all the 3 elements to fit in the space because of the way that the chart does layouts (not very dynamic) and the changeable label length and it seemed like the least of the evils at the time.